src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -635,6 +635,14 @@ def initialize_build(
         # Check degree of freedom
         assert degrees_of_freedom(blk) == 0
 
+        # For 10 and 11 effects, specific area may need a new initialization value,
+        # because the surrogate eqn for 9 and 12 effects are largely different
+        if blk.config.number_effects == 11:
+            blk.specific_area_per_m3_day.value = 3
```

Comment:
I think this is something we should still keep an issue on as a reminder and revisit, in case there is a more elegant solution to this, rather than a hard-coded value subject to a particular number of effects. Also, is it not an issue for 10 effects?

Reply:
Yea it is not elegant and I'll reconsider it. Will need to see how the model perform on the whole solution space by visualizing it as Kurby did for 12 effects.